hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/driver/impl/StateStoreZooKeeperImpl.java

```diff
@@ -63,8 +72,14 @@ public class StateStoreZooKeeperImpl extends StateStoreSerializableImpl {
       RBFConfigKeys.FEDERATION_STORE_PREFIX + "driver.zk.";
   public static final String FEDERATION_STORE_ZK_PARENT_PATH =
       FEDERATION_STORE_ZK_DRIVER_PREFIX + "parent-path";
+  public static final String FEDERATION_STORE_ZK_CLIENT_THREADS_SIZE =
+      FEDERATION_STORE_ZK_DRIVER_PREFIX + "client.size";
```

Comment:
how about changing the name to `FEDERATION_STORE_ZK_DRIVER_PREFIX + "async.max.threads"`?